docs/src/pages/components/about-the-lab/about-the-lab-zh.md

```diff
@@ -44,7 +44,7 @@ yarn add @material-ui/core@next
 ```tsx
 import '@material-ui/lab/themeAugmentation';
 
-const theme = createMuiTheme({
```

Comment:
I don't think we are supposed to do changes to the translated files. @oliviertassinari ?

Reply:
It's best to avoid it as it add noises in the pull requests. But since this one is already very noisy, no strong preferences.

Reply:
Got ya!